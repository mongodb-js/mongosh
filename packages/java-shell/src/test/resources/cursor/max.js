// before
db.coll.insert({"_id": 1, name: "Vasya"});
db.coll.insert({"_id": 2, name: "Petya"});
db.coll.insert({"_id": 3, name: "Lyusya"});
// command
db.coll.find().max({_id: 2}).hint({_id: 1});
// clear
db.coll.drop();