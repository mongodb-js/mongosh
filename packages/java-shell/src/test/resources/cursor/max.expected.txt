[ { "_id": 1, "name": "Vasya" } ]